src/names.c

```diff
@@ -0,0 +1,89 @@
+#include <glib.h>
+
+#include "names.h"
+#include "parse.h"
+
+/* Non-static as we need it for ABI compatibility, see at the end of the file */
+const char* const np_backend_to_str[NETPLAN_BACKEND_MAX_] = {
+        [NETPLAN_BACKEND_NONE] = "none",
+        [NETPLAN_BACKEND_NETWORKD] = "networkd",
+        [NETPLAN_BACKEND_NM] = "NetworkManager",
+        [NETPLAN_BACKEND_OVS] = "OpenVSwitch",
```

Comment:
```suggestion
    [NETPLAN_BACKEND_NONE] = "none",
    [NETPLAN_BACKEND_NETWORKD] = "networkd",
    [NETPLAN_BACKEND_NM] = "NetworkManager",
    [NETPLAN_BACKEND_OVS] = "OpenVSwitch",
```
Let's fix up this whitespace, while on it :)

Reply:
At some point I'll cook up a clang-format file, it'll save us both some time :)